rest-api/src/main/java/uk/gov/gchq/gaffer/rest/service/IOperationService.java

```diff
@@ -53,7 +54,7 @@
  * {@link uk.gov.gchq.gaffer.graph.Graph}.
  */
 @Path("/graph/doOperation")
-@Api(value = "/graph/doOperation", description = "Allows operations to be executed on the graph. See <a href='https://github.com/gchq/Gaffer/wiki/operation-examples' target='_blank'>Wiki</a>.")
+@Api(value = "operations", description = "Allows operations to be executed on the graph. See <a href='https://github.com/gchq/Gaffer/wiki/operation-examples' target='_blank'>Wiki</a>.")
```

Comment:
As graph/doOperation has been renamed to 'operations' you will need to update gaffer.js so that the addExampleButtons method finds the swagger sections properly.

In gaffer.js/addExampleButtons method - change 'resource_graphdoOperation' to 'resource_operations'